include/tvm/relay/attrs/nn.h

```diff
@@ -1047,12 +1047,15 @@ struct SparseTransposeAttrs : public tvm::AttrsNode<SparseTransposeAttrs> {
 /*! \brief Attributes for sparse_dense operator */
 struct SparseConv2DAttrs : public tvm::AttrsNode<SparseConv2DAttrs> {
   std::string layout;
+  int kernel_size;
```

Comment:
Suggest to follow the definition of `Conv2DAttrs` to use `Array<IndexExpr> kernel_size` here.